packages/concerto-analysis/src/comparers/properties.ts

```diff
@@ -173,16 +173,24 @@ const propertyTypeChanged: ComparerFactory = (context) => ({
         //     return;
         // }
         const versionDiff = semver.diff(aTypeVersion, bTypeVersion);
-        if (!versionDiff) {
-            return;
-        } else if (versionDiff === 'major' || versionDiff === 'premajor') {
+        if (versionDiff === 'major' || versionDiff === 'premajor') {
             context.report({
                 key: 'property-type-changed',
                 message: `The ${aType} "${a.getName()}" in the ${classDeclarationType} "${a.getParent().getName()}" changed type from "${aFQTN}" to "${bFQTN}" (type version incompatible)`,
                 element: a
             });
             return;
         }
+        // Up to this point, 'a' and 'b' have identical fully qualified type names with matching versions.
+        // Next, we verify whether their types differ locally, which would indicate aliasing.
+        if (a.getType() !== b.getType()) {
```

Comment:
```suggestion
        if (!versionDiff && a.getType() !== b.getType()) {
```
Since you are removing this condition to check where version hasn't changed, we need to cover it here I think. Because after
```js
(versionDiff === 'major' || versionDiff === 'premajor')
```
check, there are still `patch` and `minor` changes.

Reply:
Do you mean if there are `minor` or `patch` version  changes , then we should not report the `localtype` changes ?

Reply:
What I mean is the logic you are adding should not add side effects. With the current change you are not JUST checking the aliasing, but also, minor and patch changes and, you are issuing a `patch` change as a result, for both cases.

If you want to handle minor and patch changes as well, they need to be called out explicitly, with a different rule, separate from aliasing, and before the aliasing check. This is how it's done for major and premajor changes.

Up to now minor and patch changes didn't issue any analysis result. So adding them would require another discussion and would be a breaking change IMO.

Reply:
Also in your comment you are indicating that `Up to this point, 'a' and 'b' have identical fully qualified type names with matching versions.` But FQNs would be different if there are minor and patch changes.

Reply:
Ok I understand your point you are saying . But there is one issue with that, if there is a minor or patch change in the version and also some local type changes , then the logic you are saying won't report anything to the user. 
I understand completely what you meant previously that there should be specific rule for that.

Reply:
Will update the comments as well. 
I think we need a discussion on the minor and patch changes of the version. I feel even if they are not reported we should still report the localType changes . 
We can discuss this in the TechWg call